03_module/TASK3/TASK3.c

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
WARNING: Missing or malformed SPDX-License-Identifier tag in line 1
#1: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:1:
+

ERROR: trailing whitespace
#5: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:5:
+#include <linux/kernel.h> $

ERROR: trailing whitespace
#6: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:6:
+ $

WARNING: please, no spaces at the start of a line
#6: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:6:
+ $

ERROR: space required after that ',' (ctx:VxV)
#8: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:8:
+module_param(param_1,int,0660);
                     ^

ERROR: space required after that ',' (ctx:VxV)
#8: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:8:
+module_param(param_1,int,0660);
                         ^

ERROR: trailing whitespace
#9: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:9:
+ $

WARNING: please, no spaces at the start of a line
#9: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:9:
+ $

ERROR: space required after that ',' (ctx:VxV)
#11: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:11:
+module_param(param_2,int,0660);
                     ^

ERROR: space required after that ',' (ctx:VxV)
#11: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:11:
+module_param(param_2,int,0660);
                         ^

ERROR: trailing whitespace
#12: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:12:
+ $

WARNING: please, no spaces at the start of a line
#12: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:12:
+ $

WARNING: Prefer [subsystem eg: netdev]_warn([subsystem]dev, ... then dev_warn(dev, ... then pr_warn(...  to printk(KERN_WARNING ...
#15: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:15:
+	printk(KERN_WARNING "Start: param_1=%d, param_2=%d, param_1+param_2=%d\n", param_1, param_2, param_1+param_2);

ERROR: trailing whitespace
#18: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:18:
+ $

WARNING: please, no spaces at the start of a line
#18: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:18:
+ $

WARNING: Prefer [subsystem eg: netdev]_warn([subsystem]dev, ... then dev_warn(dev, ... then pr_warn(...  to printk(KERN_WARNING ...
#21: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:21:
+	printk(KERN_WARNING "Stop: param_1=%d, param_2=%d, param_1-param_2=%d\n", param_1, param_2, param_1-param_2);

ERROR: trailing whitespace
#23: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:23:
+ $

WARNING: please, no spaces at the start of a line
#23: FILE: /air/Documents/trainings/linux/kernel/pro-camp/2021/03_module/TASK3/TASK3.c:23:
+ $

total: 10 errors, 8 warnings, 30 lines checked